### PR TITLE
Update settings descriptions

### PR DESCRIPTION
**Pull request by goanpeca**
# Description

Update descriptions of fields in settings. Add a couple of helper methods on the pref dialog

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# References

# How has this been tested?

- [x] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Review by sofroniewn — APPROVED** [review]
LGTM! Thanks!!

File: napari/utils/settings/_defaults.py [hunk 1]
```diff
@@ -285,59 +277,118 @@ class ApplicationSettings(BaseNapariSettings):
...
+        title=trans._("Window state"),
+        description=trans._(
+            "Last saved state of dockwidgets and toolbars for the main window. This setting is managed by the application."
+        ),
+    )
+    window_statusbar: bool = Field(
+        True,
+        title=trans._("Show status bar"),
+        description=trans._(
+            "Toggle diplaying the status bar for the main window."
```
Comment (ppwadhwa) [inline napari/utils/settings/_defaults.py:356]: minor typo--  displaying

Comment (ppwadhwa) [inline napari/utils/settings/_defaults.py:356]:
```suggestion
            "Toggle displaying the status bar for the main window."
```

**Review by ppwadhwa — APPROVED** [review]
LGTM :)

**Review by sofroniewn — APPROVED** [review]
LGTM too!

**Pull request merged**